social-media/discord/rules-info-channel.md

```diff
@@ -0,0 +1,45 @@
+- No Private Messages - Unless confirmed publicly on the server you should never send private unsolicited messages to any members of this server. The only exception would be to report a sensitive issue to the @Admins  or @Moderators. But even then it is recommended to publicly confirm that they are okay with you sending a private message.
+- No Advertising - Please no advertisements of products or services independent of organic conversation.
+- Match your display name to what is on github. This helps for us assigning you to a team on there and allows us to contact you better for updates and concerns on contributions. This is ***NOT*** a strict requirement but it is extremely helpful in operations here.
+
+
+# Conduct
+
+https://github.com/pulsar-edit/pulsar/blob/master/CODE_OF_CONDUCT.md
+
+
+# Contributing
+
+If you are a new contributor to Pulsar and want to know how to help, feel free to interact on the server or GitHub. But if you'd like to just start programming please read our new contributor help guide. https://github.com/pulsar-edit/.github/blob/main/project-birth/CONTRIBUTING-DURING-START.md 
+
+
+# Roles
+
+Once you're done here please make your way over to ⁠role-select  and grab some roles. It helps us know who is interested and helping and who we can ping for various things that come up here as we form teams. Especially if we're not prompt enough to add you to the proper team on github. The description of these are below:
+
+These are the roles for the server in case you want to join them:
+Organizational:
+- Documentation - for our documentation. This relates to our site and all the documents we handle between it and the pulsar-edit/.github repo
+- Backend - API and server related development
+- Developer - if you're not sure what you want to contribute too or just want to contribute wherever.
+- Core - Works on the core editor and all the things that make it tick.
+- Packages - our core packages
+
+Community Related:
+- Team - these people have contributed a significant amount and have generally been a help in the community
+- Admin/Mod - these people are admins and mods bc of their help and demonstratable necessity for their place as someone to help with moderation and various other things within the community
+- Updates - when we post updates you'll get a notification. Consider this ⁠pulsar-announcements and eventually ⁠community-updates
+- Community - if you develop a package on your own outside the org and submit it grab this one! We'll try to post updates about development on this soon as this is a upcoming feature in the v2 backend!
+
+OS(for support and supporting others):
+- Windows
+- Linux
+- MacOS
+
+# Invite
+
+Invite link to the server(due to change when we are able to get a vanity link) https://discord.gg/7aEbB9dGRT
```

Comment:
```suggestion
Invite link to the server: https://discord.gg/7aEbB9dGRT
```

Personally I'd recommend we change this to leave out the vanity link mention. Only because it's not something we have ever discussed before. And even if we do decide to change it in the future we can just update it, rather than potentially warn of it's changing for an undetermined amount of time. (Would be a little awkward to have this TODO sitting out in public for say a year lol

Reply:
I just don't like random links like this. I know there are solutions to this, but I still wish we could just fully drop platforms like this somehow. But that's a discussion for elsewhere and I guess finding another temporary solution to this. Since I almost want to suggest making an alias for it under the site somehow to clean it up more? (or if we could afford a shortened version domain like pe.dev/discord)

Reply:
I agree we don't need to warn, since projects usually update these invite links as and when needed, without much ceremony. Which is fine, IMO.

Discord presents a UI about whether to join or not, when you click the link, I think? And you know it's going to be Discord, so it's not really a sketchy shortlink situation, IMO.

Reply:
It's just weird tho not having the vanity link either way. It's kinda bizarre imo that projects like us can't just have it. I'm seeing where you guys are coming from, it's just personal gripes with the systems we deal with XD